.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Report a bug
+about: File a bug to help us improve ML.NET
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**System Information (please complete the following information):**
+ - OS & Version: [e.g. Windows 10] 
+ - ML.NET Version : [e.g. ML.NET v1.5.5]
+
```

Comment:
It would also be good to know which version of .NET they are using.

Reply:
Fixed by https://github.com/dotnet/machinelearning/pull/5846/commits/6a6f2e65e58263401ee6cbcd1fdf9340e45284e0